frontend/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Is this a default readme from create-react-app?

Reply:
yes, it's the default